netlify.toml

```
@@ -1,2 +1,7 @@
 [build]
 publish = "public"
+
+[[redirects]]
+from = "/mcp"
+to = "https://docs.astro.build/en/guides/build-with-ai/"
```

```suggestion
to = "https://mcp.docs.astro.build/"
```

Why not redirect to the main landing page that as all the info/links already? instead of the full docs page? I would assume if im wanting to hit the MCP server to check if its alive, then it wouldn't redirect me to docs, but to its own hosted index.

---

Yeah I think that could make sense. Don’t need the full URL in that case, just `/` should be enough (means it would work with deploy previews etc.)
```suggestion
to = "/"
```

---

[Opinions differ on this one](https://github.com/withastro/docs-mcp/pull/1#discussion_r2267272996), but personally I'm okay with "/" as well 👍